AppKit/CPView.j

```diff
@@ -246,6 +248,9 @@ var CPViewHighDPIDrawingEnabled = YES;
 
     id                  _animator;
     CPDictionary        _animationsDictionary;
+
+    // CPViewController support
+    CPViewController _viewController @accessors(property=_viewController);
```

Comment:
I agree with you. Suprisingly cocoa has this ivar in NSView but it's better to avoid it.